filing_history_descriptions.yml

```diff
@@ -1037,3 +1037,4 @@ description:
     'registration-of-a-limited-partnership' : "**Registration** of a Limited Partnership"
     'registration-of-a-scottish-partnership' : "**Registration** of a Scottish Partnership"
     'notice-of-ceasing-to-be-a-qualifying-partnership' : "**Notice of ceasing** to be a qualifying partnership"
+    'appoint-corporate-manager-european-economic-interest-groupings' : "**Appointment** of corporate EEIG manager"
```

Comment:
Line terminated with 0. Add new line \n

Reply:
I've tested in Statler and this is not causing any issues. Agreed with Stefano not to change this